region/brinstar/green/Etecoon Energy Tank Room.json

```
@@ -2089,6 +2139,65 @@
       "name": "Base",
       "requires": [],
       "flashSuitChecked": true
+    },
+    {
+      "link": [7, 7],
+      "name": "Spikesuit - R-Mode Spark Interrupt (Gain Blue Suit)",
```

```suggestion
      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
```
The note should also be updated to reflect that the flash suit is obtained prior to this strat, no longer as part of this strat itself.

---

To follow with other flash-suit using strats, perhaps name should be "Use Flash Suit - R-Mode Spark Interrupt (Gain Blue Suit)"?

---

Even better, yeah